cmd/lotus-worker/main.go

```diff
@@ -129,6 +129,13 @@ var stopCmd = &cli.Command{
 		defer closer()
 
 		ctx := lcli.ReqContext(cctx)
+
+		// Detach any storage associated with this worker
+		err = api.StorageDetachAll(ctx)
+		if err != nil {
```

Comment:
I'm of the opinion that we should continue execution since shutdown should be best effort and should not be stopped if we can't close some paths
But the linter doesn't allow me ignore the return. Is there a better way to do this here? What are your thoughts?

Reply:
replace `err` with `_` if you want to explicitly ignore a return value. Leave a helpful comment with why you're doing so :)